printer_mods/MSiemons/TophatHingeV0.1/readme.md

```diff
@@ -0,0 +1,21 @@
+I really like the design of the tophat of the V0 but wanted to be able to open and close it easily.
+
+This is a hinge for the tophat of the V0, without changing the aesthetic.
+It uses modified panelmounts and tophat parts. Also the lower corner clips are adapted to allow for the rotation.
+All parts can be printed without support.
+
+Printed and tested: the tophat is relatively stable and stays in the open position.
+
+Requirements:
+
+4 x M3x20
+
+6 x M3x12
+
+![image](https://user-images.githubusercontent.com/38942136/126869895-856b895a-9f4e-4b0c-8633-0dcc69066b98.png)
```

Comment:
Please upload all the images into your mod directory and reference them from there.  I.e. if you upload this one as `printer_mods/MSiemons/TophatHingeV0.1/image.png`, you can reference it as

    ![image](image.png)

Reply:
Thanks for the suggestion! All images are now also in a in a folder.

Reply:
"Is Lower_Corner_adapter_2_and_4_Clip_x2.stl the same as Lower_Corner_adapted_2_and_4_Clip_x2.stl but in an unprintable orientation?"

Yes - forgot to remove it. Sorry.

Regarding the overhang: not sure how to interpret your screenshot, but Ive modified the part slightly to correct for the (very small) overhang :)

Reply:
Update on the overhang: I now realize my mistake. Ive modified the part correctly.